source/js/foundation/inject-react/pulse-tabbed-profile-filters.js

```diff
@@ -36,7 +36,8 @@ export default (apps) => {
 
     apps.push(
       new Promise((resolve) => {
-        ReactDOM.render(
+        const root = createRoot(container);
```

Comment:
Same. What is `container` referencing to?

Reply:
should also be target